demisto_sdk/commands/test_content/TestContentClasses.py

```diff
@@ -2074,66 +2074,81 @@ def execute_tests(self):
         Only to be used with server version 6.2 and above. """
+
+    # Checking configuration
     if not external_playbook_configuration:
         logger_module.info("External Playbook Configuration not provided, skipping re-configuration.")
         return False, {}, ''
+
+    # Checking server version
     server_version = get_demisto_version(client)
 
     if LooseVersion(server_version.base_version) < LooseVersion('6.2.0'):  # type: ignore
         logger_module.info("External Playbook not supported in versions previous to 6.2.0, skipping re-configuration.")
         return False, {}, ''
 
-    # if external_playbook_configuration:
     logger_module.info("External Playbook in use, starting re-configuration.")
-    restore_needed = False
 
+    # Getting current configuration
     external_playbook_id = external_playbook_configuration['playbookID']
     external_playbook_path = f'/playbook/{external_playbook_id}'
     res, _, _ = demisto_client.generic_request_func(client, method='GET',
                                                     path=external_playbook_path, response_type='object')
-    # Save Default Configuration.
+
     inputs = res.get('inputs', [])
     if not inputs:
         raise Exception("External Playbook was not found or has no inputs.")
 
+    # Save current for default Configuration.
     inputs_default = deepcopy(inputs)
     logger_module.info("Saved current configuration.")
-    """ takes external configuration of shape {"playbookID": "Isolate Endpoint - Generic V2",
-                                               "input_parameters":{"Endpoint_hostname": {"simple", "test"}}}"""
+
     changed_keys = []
     failed_keys = []
+
     # Change Configuration for external pb.
     for input_ in external_playbook_configuration["input_parameters"]:
         matching_record = list(filter(lambda x: x.get('key') == input_, inputs))
         if matching_record:
             existing_val = matching_record[0]
-            existing_val['value']["simple"] = external_playbook_configuration["input_parameters"][input_].get("simple")
-            existing_val['value']["complex"] = external_playbook_configuration["input_parameters"][input_].get("complex")
+            simple = external_playbook_configuration["input_parameters"][input_].get("simple")
+            complex = external_playbook_configuration["input_parameters"][input_].get("complex")
+
+            # If no value (simple or complex) was found, It is a typo
+            if complex is None and simple is None:
+                raise Exception(f'Did not found neither simple nor complex value for field: {input_}. '
+                                'Please check configuration valid as following: '
+                                '{<param name>: {"simple", <required value>}}')
+
+            existing_val['value']["simple"] = simple
+            existing_val['value']["complex"] = complex
```

Comment:
Why do we set it here? How is the behavior different than the way it used to be?

Reply:
I use the simple and complex in the condition above so instead of getting the value twice I get it once and use it. nothing changed in the behavior.